src/khoj/database/models/__init__.py

```diff
@@ -182,7 +182,10 @@ class ModelType(models.TextChoices):
     name = models.CharField(max_length=200, default="default")
     model_type = models.CharField(max_length=200, choices=ModelType.choices, default=ModelType.TEXT)
     bi_encoder = models.CharField(max_length=200, default="thenlper/gte-small")
-    cross_encoder = models.CharField(max_length=200, default="cross-encoder/ms-marco-MiniLM-L-6-v2")
+    bi_encoder_model_config = models.JSONField(default=dict)
+    bi_encoder_query_encode_config = models.JSONField(default=dict)
+    bi_encoder_docs_encode_config = models.JSONField(default=dict)
```

Comment:
Should we add some comments/a link explaining what these would be used for?

Reply:
Added